scripts/ramfs.go

```diff
@@ -559,32 +558,7 @@ func guessgopath() {
 		config.Gopath = gopath
 		return
 	}
-	// It's a good chance they're running this from the u-root source directory
-	log.Fatalf("Fix up guessgopath")
-	cwd, err := os.Getwd()
-	if err != nil {
-		log.Printf("GOPATH was not set and I can't get the wd: %v", err)
-		config.Fail = true
-		return
-	}
-	// walk up the cwd until we find a u-root entry. See if cmds/init/init.go exists.
-	for c := cwd; c != "/"; c = filepath.Dir(c) {
-		if filepath.Base(c) != "u-root" {
-			continue
-		}
-		check := filepath.Join(c, "cmds/init/init.go")
-		if _, err := os.Stat(check); err != nil {
-			//log.Printf("Could not stat %v", check)
-			continue
-		}
-		config.Gopath = c
-		log.Printf("Guessing %v as GOPATH", c)
-		os.Setenv("GOPATH", c)
-		return
-	}
-	config.Fail = true
-	log.Printf("GOPATH was not set, and I can't see a u-root-like name in %v", cwd)
-	return
+	log.Fatalf("You have to set GOPATH, which is typically ~/go")
```

Comment:
As of Go 1.8, "$HOME/go" is automatically used if GOPATH is not set.

Reply:
Thanks Ryan, this allows me to fix BOTH ramfs.go and bb.go.

one thing I've realized in this project how much go has changed in 6 years!

Reply:
Ah, right, I use GOPATH for a few other things, I'll leave it in for now.

Reply:
So there will be a future fix to accomodate this change.